### DISCUSSION TITLE
Zoom info for the summit: where to put it

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#83] [Team Posts]
Github gurus: @danflick and I would like to create a page with the summit info (particularly the Zoom links) such that it would be visible only to Participants. Does it sound like we need an additional repository for that then, seeing as "docs" is (and should be) world-readable? (I suppose such an additional repository whose purpose would just to contain sensitive summit-related info could just be deleted later?) An alternative would be to rely on the mailing list to distribute the Zoom link but a page which would only be accessible to Participants would be better.

**Comment by arademaker** [comment]
Yes, it looks like we will  need a repo for the summit as a dashboard to hold the sensitive information of the summit and maybe also the drafts of the notes…

**Comment by guyemerson** [comment]
I think a simpler solution would be to use a discussion for the Participants team: https://github.com/orgs/delph-in/teams/participants

**Comment by arademaker** [comment]
You are right! Good idea. We can just double check if the forum is really restricted to the participants.

**Comment by olzama** [comment]
Indeed! I forgot that discussions can be private to the team.

**Comment by emilymbender** [comment]
Since not everyone on the old ParticipantsGroup page has joined the participants team on GitHub, I think it would be good to (in parallel) re-announce on the participants mailing list that they need to, in order to get the Zoom links.

**Comment by olzama** [comment]
Yes, I think we will do the mailing list as usual; just trying to figure out if there is a convenient way to also just have a page with appropriate access.

**Comment by emilymbender** [comment]
Definitely a good idea to do both! 

(Inexplicably, Dan's message announcing the final schedule end up in my spam folder...)

**Comment by olzama** [comment]
> (Inexplicably, Dan's message announcing the final schedule ended up in my spam folder...)

Oh! For me, too! Ugh.
